TextAnalyzer/Helpers/TextFileFetcher.cs

```diff
@@ -6,31 +6,39 @@ namespace TextAnalyzer
 {
     static class TextFileFetcher
     {
-        private const string FILE_NAME = "text.txt";
+        public static string FILE_NAME = "text.txt";
+        private const string REPORT_FILE = "report.txt";
         private const string LINK = "https://s3.zylowski.net/public/input/3.txt";
 
+        public static bool DownloadFileFromURL(string URL, string fileName = null)
+        {
+            using (var client = new WebClient())
+            {
+                try
+                {
+                    client.DownloadFile(URL, fileName != null ? fileName: Path.GetFileName(URL) );
+                    return true;
+                }
+                catch (Exception)
+                {
+                    return false;
+                }
+            }
+        }
+
         /// <summary>
         /// Get Text File from web client.
         /// </summary>
         /// <returns>Returns the string with entire text.</returns>
-        public static string GetTextFileString()
+        public static string GetTextFileString(string link = LINK, string filename = null)
```

Comment:
Wyciagnalbym te pola na gore, a nawet utworzyl region : fields 😊  const

Reply:
To tylko zmiana kosmetyczna , nie konieczne:P

Reply:
Nie do końca rozumiem, chcesz wyciągnąć parametry z metody i całkowicie je zastąpić polami? Można przedyskutować z Piotrem czy te parametry są potrzebne i ewentualnie zaimplementować jako mały oddzielny refactor.